<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>619</width>
       <height>519</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddFolder"/>
   <addaction name="actAddFile"/>
   <addaction name="actZoomIn"/>
   <addaction name="actZoomOut"/>
   <addaction name="actZoomRealSize"/>
   <addaction name="actZoomFitW"/>
   <addaction name="actZoomFitH"/>
   <addaction name="actDelNode"/>
   <addaction name="actQuit"/>
   <addaction name="actScanNode"/>
   <addaction name="actDockVisible"/>
   <addaction name="actDockFloat"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>338</width>
     <height>45</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>491</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>331</width>
       <height>0</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <action name="actAddFolder">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/addFloder.png</normaloff>:/images/Icon/addFloder.png</iconset>
   </property>
   <property name="text">
    <string>添加目录...</string>
   </property>
  </action>
  <action name="actAddFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/newfile.png</normaloff>:/images/Icon/newfile.png</iconset>
   </property>
   <property name="text">
    <string>添加文件...</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/zoom in.png</normaloff>:/images/Icon/zoom in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actZoomOut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/zoom out.png</normaloff>:/images/Icon/zoom out.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/zoom.png</normaloff>:/images/Icon/zoom.png</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/fit-width.png</normaloff>:/images/Icon/fit-width.png</iconset>
   </property>
   <property name="text">
    <string>合适宽度</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/fit-height.png</normaloff>:/images/Icon/fit-height.png</iconset>
   </property>
   <property name="text">
    <string>合适高度</string>
   </property>
  </action>
  <action name="actDelNode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/delete.png</normaloff>:/images/Icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/exit.png</normaloff>:/images/Icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actScanNode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/scan.png</normaloff>:/images/Icon/scan.png</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/visible.png</normaloff>:/images/Icon/visible.png</iconset>
   </property>
   <property name="text">
    <string>窗体可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Icon/Float.png</normaloff>:/images/Icon/Float.png</iconset>
   </property>
   <property name="text">
    <string>窗体浮动</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
